### PR TITLE
Create Search Page

### LINKED ISSUE #12: [HTML-CSS] Create search page
Adding the required html and css for the search page

### PR DESCRIPTION
**Pull request by Yousef-Dall**
relates #12

### CODE REVIEW

**Review by basillubbad — APPROVED** [review]

File: searchPage/search.html [hunk 1]
```diff
@@ -1,24 +1,81 @@
...
+      </nav>
     </header>
-    <script src="search.js"></script>
 
-</body>
-</html>
\ No newline at end of file
+    <div id="serach-container">
+      <div id="ima">
+        <img id="image" src="../img/Group 13.png" alt="" />
```
Comment (yaqootturman) [inline searchPage/search.html]: the image name is not camelCase as naming convention you follow

File: searchPage/search.html [hunk 2]
```diff
@@ -1,24 +1,81 @@
...
+      <div id="input-field">
+        <i id="search-icon" class="fas fa-search icon"></i>
+        <input
+          list="countries"
+          id="input"
+          name=""
+          placeholder="Enter country name"
+        />
+
+        <datalist id="countries">
```
Comment (yaqootturman) [inline searchPage/search.html:42]: the list appears when the user clicks on the input field which I think should change, specially it appears with dark color and above countries pictures..
another suggestion: you can make a 'list sign' appear as a default and clicking on it lead to see the list

File: searchPage/search.css [hunk 3]
```diff
@@ -1,27 +1,152 @@
...
+}
+
+#ima {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 0px;
+}
+
+#most {
```
Comment (yaqootturman) [inline searchPage/search.css:41]: the size of the font is big, and you need to decrease left margin and increase the top one

**Review by yaqootturman — CHANGES_REQUESTED** [review]

**Review by nada-abuzaid — APPROVED** [review]

**Review by yaqootturman — APPROVED** [review]
You didn't change the top margin, I approved the PR

**Pull request merged**